src/coreclr/tools/r2rtest/Commands/CompileNugetCommand.cs

```diff
@@ -58,7 +58,9 @@ public static int CompileNuget(BuildOptions options)
                     string appFolder = Path.Combine(nugetOutputFolder, $"{package}.TestApp");
                     Directory.CreateDirectory(appFolder);
 
-                    int exitCode = DotnetCli.New(appFolder, "console", nugetLog);
+                    Version version = Environment.Version;
+                    string targetFramework = $"net{version.Major}.{version.Minor}";
```

Comment:
@kasperk81 @elinor-fung if you folks have any recommendations about this variable name I'm all ears.